app/src/main/java/com/keylesspalace/tusky/util/OkHttpUtils.java

```diff
@@ -121,47 +81,6 @@ private static Interceptor getUserAgentInterceptor() {
         };
     }
 
-    /**
```

Comment:
We don't need this anymore because we just bundle the provider, right?

Reply:
yes